src/services/car/messageLead.ts

```diff
@@ -55,3 +62,31 @@ export const sendMessageLead = async ({
     })
   }
 }
+
+export const fetchDealerMessageLeads = async ({
+  dealerId,
+  page = 0,
+  size = 7,
+  options = {},
```

Comment:
When fetching listings, we pass the search query (sorting, pagination and filters) as a single object. Can we do the same here?